modules/document_repository/php/document_repository.class.inc

```diff
@@ -78,10 +78,9 @@ class Document_Repository extends \NDB_Menu_Filter
                           'v.Data_dir',
                           'v.Date_uploaded',
                          );
-
-        $openAccordion = $_GET['openAccordion'];
+        $openAccordion = isset($_GET['openAccordion']) ?? false;
```

Comment:
and this request for change also still valid https://github.com/aces/Loris/pull/3632/files/f93b152ec814bd2b246164b752c3ea2b703733a3..7b414f0e7f5a60132c8579ac6fdf0d75b4eed867#r194892059